tests/test_operation.py

```diff
@@ -321,6 +321,21 @@ class DummyOp(Operator):
         assert op._ndim_params == (ndim_params,)
         assert op.ndim_params == (0,)
 
+    def test_expand_fn_deprecated(self):
```

Comment:
Can you also add a docstring for the test

```suggestion
    def test_expand_deprecated(self):
```